software/proposals/cough_analysis.md

```diff
@@ -0,0 +1,60 @@
+---
+path: "/projects/software/url_to_be_created"
+title: "Cough Analysis"
+---
+
+| | |
+|-|-|
+| Proposal Stage |   Draft   |
+| Tags           |   Audio, Analysis, Diagnosis, Cough   |
+| Created Date   |   18/03/2020   |
+| Lead           |   Dave Moffat   |
+| Slack Channel  |   proj-dev-ai-cough-detection   |
```

Comment:
I couldn't find this channel, is it valid?

Reply:
Yes, this is the channel. It is on the open-covid19 Slack project

Reply:
are we talking about the same Slack organization? helpfulengineering.slack.com ?
I couldn't find any. You can ping me privately on Slack if you prefer to sort it out

Reply:
No, we are not. I am talking about open-covid19.slack.com

Reply:
This repo belongs to the HelpfulEngineering one. I think you pushed it to the wrong place 🙂
What is the goal with this PR? Is the project a work in progress thing that you are managing separately?
If you want to migrate it, feel invited to join our Slack, get some volunteers around the idea, etc